packages/maker/appx/README.md

```diff
@@ -0,0 +1,18 @@
+## Electron Forge: Maker APPX
```

Comment:
```suggestion
## maker-appx
```

Bikeshedding here a bit: I'm not sure I like the use of full words and punctuation where the grammar doesn't quite make sense.

If we're only displaying these readmes within the Forge API docs, then I don't think we need the reference `@electron-forge` in the title. I also think having the verbatim package name `maker-appx` is a bit cleaner

Reply:
I think that's fair! I was honestly waffling back and forth on this one, because the README also shows up for the npm package. However, I think it'd be rare for a user to be looking at that package outside of Forge itself. I'll make the changes to each README to just use the package name 👍